### PR TITLE
feat: compute_hyperdisk_create_from_pool

### PR DESCRIPTION
**Pull request by gryczj**
## Description

Fixes #<ISSUE-NUMBER>

Note: Before submitting a pull request, please open an issue for discussion if you are not associated with Google.

## Checklist
- [x] I have followed guidelines from [CONTRIBUTING.MD](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md) and [Samples Style Guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [ ] **Tests** pass:   `npm test` (see [Testing](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#run-the-tests-for-a-single-sample))
- [x] **Lint** pass:   `npm run lint` (see [Style](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#style))
- [ ] These samples need a new **API enabled** in testing projects to pass (let us know which ones)
- [ ] These samples need a new/updated **env vars** in testing projects set to pass (let us know which ones)
- [x] This pull request is from a branch created directly off of `GoogleCloudPlatform/nodejs-docs-samples`. Not a fork.
- [ ] This sample adds a new sample directory, and I updated the [CODEOWNERS file](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CODEOWNERS) with the codeowners for this sample
- [ ] This sample adds a new sample directory, and I created [GitHub Actions workflow](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/CONTRIBUTING.md#adding-new-samples) for this sample
- [ ] This sample adds a new **Product API**, and I updated the [Blunderbuss issue/PR auto-assigner](https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/main/.github/blunderbuss.yml) with the codeowners for this sample
- [x] Please **merge** this PR for me once it is approved

### CODE REVIEW

**Comment by gryczj** [comment]
Looks like the source projects/debian-cloud/global/images/family/debian-10 does not exist. I changed the path to use debian-11 and that solved the problem. I can move these changes to separate PR and mark as refactor if there is a need.

File: compute/test/createComputeHyperdiskFromPool.test.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  const storagePoolName = 'storage-pool-name';
+  const disksClient = new DisksClient();
+  let projectId;
+
+  before(async () => {
+    projectId = await disksClient.getProjectId();
+  });
+
+  after(async () => {
+    await deleteDisk(disksClient, projectId, zone, diskName);
```
Comment (subfuzion) [inline compute/test/createComputeHyperdiskFromPool.test.js:41]: Hi @gryczj There's no real value in putting the delete logic into a separate module and importing it. I recommend removing `util.js` and just delete the resource here and let it throw; if it throws, the test will fail right here in the `after` hook (and that's a good thing). Thanks!

**Review by subfuzion — CHANGES_REQUESTED** [review]
The PR looks good, ready to merge after you make the requested change. Thank you!

File: compute/test/createComputeHyperdiskFromPool.test.js (hunk 1 above)
Comment (gryczj) [reply compute/test/createComputeHyperdiskFromPool.test.js:41]: changes applied also to other samples- which were merged yesterday

**Review by subfuzion — APPROVED** [review]

**Pull request merged**